lib/views/pages/profile/posts_info/info_card_post.dart

```diff
@@ -23,28 +26,41 @@ class InfoCardPost extends StatelessWidget {
       height: 80,
       decoration: BoxDecoration(
         color: Theme.of(context).colorScheme.secondaryContainer,
-        borderRadius: const BorderRadius.all(Radius.circular(10)),
+        borderRadius: borderRadius,
         boxShadow: [shadow],
       ),
-      child: Center(
-        child: ListTile(
-          title: Text(
-            title,
-            style: Theme.of(context).textTheme.titleSmall,
-          ),
-          subtitle: Text(
-            description,
-            style: Theme.of(context).textTheme.bodySmall,
-            maxLines: 2,
-            overflow: TextOverflow.ellipsis,
+      child: Material(
+        clipBehavior: Clip.hardEdge,
+        borderRadius: borderRadius,
+        color: Colors.transparent,
+        child: InkWell(
+          onTap: () => showDialog<void>(
+            context: context,
+            builder: (BuildContext context) {
+              return PostPopUp(title: title, description: description);
+            },
           ),
-          trailing: const IconButton(
-            icon: Icon(
-              Icons.delete,
-              size: 32,
+          child: Center(
+            child: ListTile(
+              title: Text(
+                title,
+                style: Theme.of(context).textTheme.titleSmall,
```

Comment:
Nice job ! The transition to the PoPup looks good to me :+1: 
I just think that we can force the post title to be on a single line here to avoid an overflow if it is too long.

```suggestion
                title,
                maxLines: 1,
                overflow: TextOverflow.ellipsis,
                style: Theme.of(context).textTheme.titleSmall,
```

Reply:
Good Idea !